src/app/header/components/search-bar/search-bar.component.scss

```diff
@@ -1,16 +1,18 @@
+$greenWalla: #13C1AC;
+
 .SearchBar {
     display: flex;
     flex-direction: row;
     justify-content: center;
     width: 100%;
     height: 40px;
+
     form {
         margin-right: 16px;
         padding: 4px;
         padding-left: 8px;
         border-radius: 8px;
-        outline-style: solid;
-        outline-color: #13C1AC;
+        border: 2px solid $greenWalla;
```

Comment:
Could you explain why did you use the border instead of the outline please??

Reply:
Oh yeah, sorry, so from what I understood, the outline takes place outside the component, so it might overlap other elements. That would be the main reason, border is more controllable.